app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+  <header class="navbar">
+    <nav class="nav-items">
+      <%= link_to t('log'), journal_entries_path, class: "nav-link" %>
+      <%= link_to t('tasks'), tasks_path, class: "nav-link" %>
+    </nav>
+    <%= link_to root_path do %>
+      <div class="logo"> Journey.</div>
```

Comment:
i18n please

Reply:
no it won't but it is still being exposed as a string and as said earlier we will have all labels in i18n and not raw strings on our code

Reply:
ok thank you for clarifyeing